docs/src/main/asciidoc/cdi.adoc

```diff
@@ -188,10 +188,26 @@ A: You can use all the built-in scopes mentioned by the specification except for
 
 NOTE: There can be other custom scopes provided by Quarkus extensions. For example, `quarkus-narayana-jta` provides `javax.transaction.TransactionScoped`.
 
+== _Q: `@ApplicationScoped` and `@Singleton` look very similar. Which one should I choose for my Quarkus application?_
```

Comment:
Maybe also add a sentence about accessing fields of a bean, i.e. if such an operation is really needed, users should stick with `@Singleton`

Reply:
That's a good point!